components/brave_rewards/resources/tip/index.html

```diff
@@ -6,6 +6,7 @@
 <title>Rewards Tip</title>
 <link rel="stylesheet" href="chrome://resources/brave/css/reset.css">
 <link rel="stylesheet" href="chrome://resources/brave/fonts/poppins.css">
+<link rel="stylesheet" href="chrome://resources/brave/leo/css/variables.css">
```

Comment:
Can you check the actual tip panel to make sure things look OK?

Reply:
How do I trigger the tip panel?

Reply:
I can try this branch out today.

Reply:
Thanks @zenparsing!